src/main/java/org/prebid/cache/config/StorageConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.prebid.cache.config;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Configuration
```

Comment:
shouldn't it work without `@Configuration`?

Reply:
No, for that to work without `@Configuration` we need to change component scan method.